build-static.sh

```diff
@@ -76,18 +81,26 @@ else
   docker_image=alpine:$DTAG
 fi
 
+info $docker_image
+
 # Copy to the build directory
 cp -r $DIR/source/$PKG/* $PKGDIR
 cp -r $DIR/lib $PKGDIR
 
+if [ -d $DIR/build ]; then
+mkdir $PKGDIR/local_builds
+cp -r $DIR/build/*${TARGET_ARCH}*.tar.xz $PKGDIR/local_builds || :
+cp $DIR/build/SHA512SUMS $PKGDIR/local_builds || :
+fi
```

Comment:
I suggest to change the logic of copying (starting at line 86), to mount `$DIR/source/$PKG/*` and `$DIR/lib` in read-only into `$CONTAINERDIR`, and `build` as read-write.

Just make sure to check if `SHA512SUMS` isn't modified by using `sha512sum SHA512SUMS` before and after entering into the container.